mindsdb/integrations/handlers/couchbasevector_handler/couchbasevector_handler.py

```diff
@@ -366,10 +366,12 @@ def create_table(self, table_name: str, if_not_exists=True):
         """
         cluster = self.connect()
         bucket = cluster.bucket(self.bucket_name)
-        scope = bucket.scope(self.scope)
-        _ = scope.collection(table_name)
+        scope = bucket.scope(self.scope)  
         try:
-            bucket.collections().create_collection(scope_name="color", collection_name=table_name)
+            bucket.collections().create_collection(
+                scope_name=self.scope, 
```

Comment:
Should't this be the `scope` ?

Reply:
fixed it @ZoranPandovski

Reply:
Sorry this was a question about the value for the `scope_name`. Do we need the self.scope or scope object?

Reply:
i think we need self.scope 
this sets self.scope as either the scope name from connection data or default